### ISSUE TITLE
v6.0.2 Logs are accepted, but actually not stored in Loki

### ISSUE THREAD

**Issue by braxnu** [JaniAnttonen/winston-loki#80]
My setup:

```
docker run -d -p 3100:3100 grafana/loki
```

```js
import winston from 'winston'
import Loki from 'winston-loki'

const logger = winston.createLogger({
  transports: [
    new Loki({
      host: 'http://127.0.0.1:3100',
    }),
  ],
})

logger.info("test")
```

The JSON payload sent to Loki in v6.0.2 contains "[Object object]" instead of the actual labels payload. This was introduced by exactly this change https://github.com/JaniAnttonen/winston-loki/pull/79/commits/ec82e7997cef4fbd291989062cf31330b3b7f3a1

The payload sent to Loki is:
```json
{
  "streams": [
    {
      "stream": "[object Object]",
      "values": [
        [
          "1634077254005000000",
          "test"
        ]
      ]
    }
  ]
}
```

There is no http error response, but these are impossible to find in grafana.

**Comment by SilSalADRESYS** [comment]
We ran into an issue too on one application which was using v6.0.2; basically, the issue ran as follows:

- The batcher.js, if you log the response from its post request (and NOT the error) in winston-loki/src/batcher.js line 195, you will actually get back an error from the Go Parser used by Loki, in that there is a malformed body.
- We found that we kept getting error 400 responses in the access logs for our nginx setup in front of Loki (but no error logs), so basically - the errors weren't being returned as errors from Loki, so the winston-loki package thought it was okay. Hence why it never showed up in the Node application logs.

The specific error it got back from the Loki parser is "Value looks like object, but can't find closing '}' symbol", source code here: https://github.com/buger/jsonparser/blob/master/parser.go
It seemed to always point at byte #10 at the end of the log message, which was in the context/metadata object. Fiddling with the Winston formatter settings didn't seem to have an impact.

Reverting to winston-loki v6.0.1 fixed the problem. Hope this helps!
